pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -54,6 +56,8 @@ const (
 	permit                                    = "Permit"
 )
 
+var configDecoder = scheme.Codecs.UniversalDecoder()
```

Comment:
is the new logic introduced to this file tested somewhere indirectly?

Reply:
I initially thought it would be close to impossible because we don't preserve the args, but I just mocked the registry :)